lib/hanami/config.rb

```diff
@@ -104,7 +104,11 @@ class Config
     # @!attribute [rw] no_auto_register_paths
     #   Sets the paths to skip from container auto-registration.
     #
-    #   Defaults to `["entities"]`.
+    #   Paths are relative to the app root or the slice root.
+    #
+    #   The `/lib` directory is always excluded from auto-registration.
```

Comment:
If we clarify the `/app` detail above, I wonder whether this line is needed anymore? I think this kind of information is better suited to our guides, vs API-level docs for an individual setting?